### PR TITLE
Polish docs of AVC test check

### PR DESCRIPTION
**Pull request by happz**
Pull Request Checklist

* [x] implement the feature

### CODE REVIEW

File: tmt/checks/avc.py [hunk 1]
```diff
@@ -282,6 +282,18 @@ class AvcDenials(CheckPlugin[Check]):
         check:
           - name: avc
 
+    .. note::
+
+        To work correctly, check requires SELinux to be enabled on the
```
Comment (martinhoyer) [inline tmt/checks/avc.py:287]:
```suggestion
        To work correctly, the check requires SELinux to be enabled on the
```
or `this` or `avc check` or..?

**Review by martinhoyer — APPROVED** [review]
Looks good to me, just minor nitpick.
Also thinking whether it would make sense to mention AVC means Access Vector Cache, but probably nah?

**Review by thrix — APPROVED** [review]
LGTM

**Review by psss — APPROVED** [review]
Nice, thanks!

**Comment by psss** [comment]
Docs change & docs building passed --> merging.

**Pull request merged**